File: my-submissions/0167-two-sum-ii---input-array-is-sorted/solution.js
/**
 * @param {number[]} numbers
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (arr, target) {
    let i = 0;
    let j = arr.length - 1;

    while (arr[i] + arr[j] !== target) {
        let sum = arr[i] + arr[j];
        if (sum > target) {
            j--;
        }
        else {
            i++;
        }

    }
    return [i + 1, j + 1]
};

File: my-submissions/0167-two-sum-ii---input-array-is-sorted/solution.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int i=0;
        int j=numbers.length-1;
        while(numbers[i]+numbers[j]!=target){
            int sum=numbers[i]+numbers[j];
            if(sum>target)j--;
            else i++;
        }
        return new int[] {i+1,j+1};
    }
}
